app/views/shared/_navbar.html.erb

```diff
@@ -48,7 +48,7 @@
   <div class="navbar-wagon-second hidden-xs hidden-sm font-oswald">
     <div>
       En ce moment :
-      <% topics.each do |topic| %>
```

Comment:
you don't need to use the instance variable `@topics` if you call the partial with `<%= render 'navbar', topic: @topic %>`